src/components/onboarding/steps/Step3.vue

```diff
@@ -19,6 +19,53 @@
       People can directly support your content, cause project<br>
       (or you for simply being you) with no middleman.
     </p>
+
+    <p class="highlited">
+      To start, you'll need a wallet. It's free. It's easy.<br>
+      <span class="important">
+        1. Click a browser extensions or mobile app
```

Comment:
This `1.` when there is no 2 is very confusing

Reply:
It was the wording provided from comms i guess.

Reply:
Yes, I think it is just a typo from the comms

Reply:
Yes. This is how the content.